### ISSUE TITLE
Build issue on ghc 7.10

### ISSUE THREAD

**Issue by dagit** [matterhorn-chat/mattermost-api#36]
Looks like the travis build on ghc 7.10 is failing in the websocket package:

```
src/Network/WebSockets/Hybi13.hs:23:62:
    Module ‘Data.Binary.Get’ does not export ‘getInt64be’
Failed to install websockets-0.11.2.0
```

I don't know if this is an issue with the way our dependencies are specified or if websockets just doesn't work with 7.10 anymore. I do suspect that the issue can be resolved by being more restrictive about the version of binary that we use allow on 7.10.

**Comment by jtdaugherty** [comment]
This turns out to be because `websockets` uses a very loose lower bound on `binary` and allows many versions of `binary` that don't include `getInt64be`. I've submitted a fix to them: https://github.com/jaspervdj/websockets/pull/155. I've also installed a temporary `binary` bound in this package to avoid the issue until `websockets` releases a fix: 2f5fc1f22ca2fad4c744179ba4d15e4a7dde628e. With that I was able to build with `7.10.1`.
